README.md

```diff
@@ -1,17 +1,35 @@
-# Miranum-Website
+# Miranum Documentation 
 
-This repository contains the code our website is build with.
-The life version can be found under [https://miranum.com/](https://miranum.com/)
+This repository contains the Miranum documentation with can be found under [https://miranum.com/](https://miranum.com/)
 
-## development
+## Documentation Guidelines
+### PRs for every change
+All changes have to be done in a separate Branch. As soon as the changes are done please open a PR. A GitHub Action runs 
+with every commit to a Branch and checks if the documentation can be build. If you create a new branch make sure to name him according 
+to what you to (e.g. feat/xyz or fix/xyz). Please use semantic commit messages as described [here](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716).
 
-You'd like to contribute? You're welcome!
+### Structure
+Name Markdown files according to the title. This makes it easier to find a file.
+Avoid non-alphanumeric characters in titles. Use the file name as internal document id to reference in the appropriate sidebars file.
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+### Style Guide 
+We will be using the writing [style guide defined by Camunda](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/technical-writing-styleguide.md).
+It outlines writing techniques and practices to ensure uniform styling across documentation and to yield a more cohesive and organized user experience.
+We also want to ensure a unified naming of our product components. Hence, please use **Miranum Connect**, **Miranum IDE**, **Miranum: Modeler**, **Miranum: Form Builder** and **Miranum: Console** if you are referring to them.  
```

Comment:
Should we name the VS-Code Extension with `:`?

```suggestion
We also want to ensure a unified naming of our product components. Hence, please use **Miranum Connect**, **Miranum IDE**, **Miranum Modeler**, **Miranum Form Builder** and **Miranum Console** if you are referring to them. 
```

Reply:
Good point @lmoesle - I have thought about this and we might want to talk about it. Imho it is important to have consistent naming. If we are not using the ":" we should remove it from our VS Code Plugins as well which I do not think we want to... 
(But I do agree it does not look to nice in the documentation)

Reply:
Well we are still at an early stage. Renaming the extensions may be unpleasant but it's better to do it now than later. If you have a good suggestion I could rename the extension we currently have and add the naming conventions to our tech docs.

Reply:
I would say that we should not use any special characters like ":" in the product names. I am fine with Miranum Modeler and co. 👍